packages/anubis/PKGBUILD

```diff
@@ -0,0 +1,48 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=anubis
+_pkgname=anubis-netsec
+pkgver=1.1.3.r0.g9ea89fa
+pkgrel=1
+pkgdesc='Subdomain enumeration and information gathering tool.'
+arch=('any')
+groups=('blackarch' 'blackawrch-scanner' 'blackarch-recon')
+url='https://github.com/jonluca/anubis'
+license=('MIT')
+depends=('python' 'python-python-nmap' 'python-shodan' 'python-docopt'
+         'python-requests' 'python-censys' 'python-dnspython')
+makedepends=('git' 'python-build' 'python-pip')
+source=("git+https://github.com/jonluca/$pkgname.git")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  git describe --long --tags | sed 's/\([^-]*-g\)/r\1/;s/-/./g'
```

Comment:
There are git tags https://github.com/jonluca/Anubis/tags so you should use https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-generic-git#L24 instead

Reply:
Isn't this exactly L24 of that template?